Gemfile

```diff
@@ -11,4 +11,6 @@ gem "asciidoctor-pdf", "~> 1.5.beta.7"
 gem "concurrent-ruby", "~> 1.1"
 gem "rouge", "~> 3.3"
 gem "asciidoctor-epub3", "~> 1.5.alpha.9"
-gem "kindlegen", "~> 3.0"
+# TODO: uncomment when issue(https://github.com/tdtds/kindlegen/issues/42) solved
+# gem "kindlegen", "~> 3.0"
+gem 'json'
```

Comment:
I would prefer YAML which [included in stdlib](https://ruby-doc.org/stdlib-2.7.0/libdoc/yaml/rdoc/YAML.html) but I'm ok to use JSOn if it works

Reply:
I agree with you. It's just that JSON came to my mind first.